### PR TITLE
Expose segmentation tags in moderation

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: kitsune/questions/views.py [hunk 1]
```diff
@@ -1424,17 +1447,27 @@ def _answers_data(request, question_id, form=None, watch_form=None, answer_previ
...
+    If tag_ids is provided, adds tags with those IDs to the question.
+    Otherwise looks for tag name in request.POST['tag-name'].
 
+    Returns a tuple of (question, tag_names) if successful.
+    Returns (None, None) if no valid tags found or SumoTag.DoesNotExist raised.
     """
+
+    question = get_object_or_404(Question, pk=question_id)
+    if tag_ids:
+        sumo_tags = SumoTag.objects.filter(id__in=tag_ids)
```
Comment (akatsoulas) [inline kitsune/questions/views.py:1464]: This should be limited only to segmentation tags

File: kitsune/flagit/jinja2/flagit/content_moderation.html [hunk 2]
```diff
@@ -7,7 +7,12 @@
         <hgroup>
           <h2 class="sumo-card-heading">{{ _('Flagged {t} (Reason: {r})')|f(t=object.content_type, r=object.get_reason_display()) }}</h2>
           {% if object.notes %}
-            <p class="notes">{{ _('Other reason:') }} {{ object.notes }}</p>
+            {% if object.content_type.model == 'question' %}
+
```
Comment (escattone) [inline kitsune/flagit/jinja2/flagit/content_moderation.html:11]: Nit. Remove this empty line?

File: kitsune/flagit/jinja2/flagit/content_moderation.html [hunk 3]
```diff
@@ -7,7 +7,12 @@
         <hgroup>
           <h2 class="sumo-card-heading">{{ _('Flagged {t} (Reason: {r})')|f(t=object.content_type, r=object.get_reason_display()) }}</h2>
           {% if object.notes %}
-            <p class="notes">{{ _('Other reason:') }} {{ object.notes }}</p>
+            {% if object.content_type.model == 'question' %}
+             
+              <p class="notes">{{ _('Additional notes:') }}<a target="_blank" href="{{ object.content_object.get_absolute_url() }}">{{ object.notes }}</a></p>
```
Comment (escattone) [inline kitsune/flagit/jinja2/flagit/content_moderation.html:12]: Nit. Add space before link.

image

File: kitsune/sumo/static/sumo/js/flagit.js [hunk 4]
```diff
@@ -1,123 +1,151 @@
...
-                        if (updateStatusSelect && updateStatusSelect.tagName === 'SELECT') {
-                            updateStatusSelect.value = '1';
-                        }
+                // Update tags
+                if (this.classList.contains('tag-select')) {
+                    const selectedTags = Array.from(this.selectedOptions).map(option => option.value);
+                    const url = `/en-US/questions/${questionId}/add-tag-async`;
+                    const response = await fetchData(url, { method: 'POST', body: { tags: selectedTags } });
+                    if (response) {
+                        console.log(response.message);
```
Comment (escattone) [inline kitsune/sumo/static/sumo/js/flagit.js:93]: Nit. Is this a leftover from debugging?

**Review by escattone — APPROVED** [review]
Just some nits to consider, but it looks good! Thanks @akatsoulas!

File: kitsune/questions/views.py (hunk 1 above)
Comment (akatsoulas) [reply kitsune/questions/views.py:1464]: Added a custom manager to return non-archived tags that start with `seg-`

**Pull request merged**